Resources/ArmTemplates/parameters.json

```diff
@@ -259,6 +259,18 @@
         },
         "PreviewRedirectUri:1:ContentType": {
             "value": "__ContentType1__"
+        },
+        "PreviewRedirectUri:2:RedirectUri": {
```

Comment:
It makes more sense to change 2 to footer?

Reply:
The values have to be written like this so it can be compiled into a proper list in the appsettings and can be easily retrieved from the app without writing specific configuration retrieval for each content type